src/main/kotlin/app/batch/Config.kt

```diff
@@ -1,8 +1,8 @@
 package app.batch
 
+import app.utils.logging.*
```

Comment:
Since we are importing `val`s it might be easier to read to import them explicitly?

Reply:
We're using 5, my threshold for "*" is usually 3 :-) For one or two, totally.